app/src/pages/inside/testCaseLibraryPage/createTestCaseModal/useCreateTestCase.ts

```diff
@@ -24,10 +24,12 @@ import { useDebouncedSpinner } from 'common/hooks';
 import { URLS } from 'common/urls';
 import { hideModalAction } from 'controllers/modal';
 import { showErrorNotification, showSuccessNotification } from 'controllers/notification';
-import { getTestCasesAction } from 'controllers/testCase';
+import { getTestCasesAction, Folder } from 'controllers/testCase';
+import { createFoldersSuccessAction } from 'controllers/testCase/actionCreators';
 
 import { CreateTestCaseFormData, ManualScenarioType } from './createTestCaseModal';
 import { messages } from './basicInformation/messages';
+import isString from 'es-toolkit/compat/isString';
```

Comment:
```suggestion
import { isString } from 'es-toolkit/compat';
```
And also let's move to already grouped third-party imports above

Reply:
done